### PR TITLE
Fixed some typos in README and removed e.g. from example usage

### PR DESCRIPTION
**Pull request by kamperh**
Changed the example code snippets so that they can be copy-and-pasted directly, and fixed some typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,26 +38,29 @@ Leader-board for the ZeroSpeech 2020 challenge can be found [here](https://zeros
...
+    Example usage:
     ```
-    e.g. python preprecess.py in_dir=../datasets/2020/2019 dataset=2019/english
+    python preprocess.py in_dir=../datasets/2020/2019 dataset=2019/english
     ```
     
 ## Training
    
-1.  Train the VQ-CPC model (pretrained weights will be released soon):
+1.  Train the VQ-CPC model (pretrained weights are given above):
```
Comment (bshall) [inline README.md]: If you don't mind, could we change this to:
```suggestion
1.  Train the VQ-CPC model (or download pretrained weights [here](https://github.com/bshall/VectorQuantizedCPC/releases/tag/v0.1):
```

File: README.md [hunk 2]
```diff
@@ -80,8 +83,9 @@ Note: the `synthesis list` is a `json` file:
 containing a list of items with a) the path (relative to `in_dir`) of the source `wav` files;
 b) the target speaker (see `datasets/2019/english/speakers.json` for a list of options);
 and c) the target file name.
+Example usage:
 ```
-e.g. python convert.py cpc_checkpoint=checkpoints/cpc/english2019/model.ckpt-25000.pt vocoder_checkpoint=checkpoints/vocoder/english2019/model.ckpt-150000.pt in_dir=../datasets/2020/2019 out_dir=submission/2019/english/test synthesis_list=datasets/2019/english/synthesis.json in_dir=../../Datasets/2020/2019 dataset=2019/english
+python convert.py cpc_checkpoint=checkpoints/cpc/english2019/model.ckpt-25000.pt vocoder_checkpoint=checkpoints/vocoder/english2019/model.ckpt-150000.pt in_dir=../datasets/2020/2019 out_dir=submission/2019/english/test synthesis_list=datasets/2019/english/synthesis.json in_dir=../../Datasets/2020/2019 dataset=2019/english
 ```
 Voice conversion samples will be available soon.
```
Comment (bshall) [inline README.md]: Might as well add another link to the samples too? It's nice to have the links near to the different sections as well.
```suggestion
Voice conversion samples are available [here](https://bshall.github.io/VectorQuantizedCPC/)
```

**Review by bshall — CHANGES_REQUESTED** [review]

**Comment by bshall** [comment]
Thanks for the suggestions @kamperh. If you don't mind making a couple of small changes, that's be great :)

File: README.md [hunk 3]
```diff
@@ -87,7 +88,8 @@ Example usage:
 ```
 python convert.py cpc_checkpoint=checkpoints/cpc/english2019/model.ckpt-25000.pt vocoder_checkpoint=checkpoints/vocoder/english2019/model.ckpt-150000.pt in_dir=../datasets/2020/2019 out_dir=submission/2019/english/test synthesis_list=datasets/2019/english/synthesis.json in_dir=../../Datasets/2020/2019 dataset=2019/english
 ```
-Voice conversion samples will be available soon.
+Voice conversion samples are available
+[here](https://bshall.github.io/VectorQuantizedCPC/).
```
Comment (kamperh) [inline README.md:92]: I'm checking out how to leave comments :)

File: README.md (hunk 2 above)
Comment (kamperh) [reply README.md]: I've made the changes below (I didn't realise you could commit it directly!).

**Review by bshall — APPROVED** [review]

**Pull request merged**